tests/integration/requirements.txt

```
@@ -1,4 +1,4 @@
-ansible-core>=2.18.0,<2.19.0
+ansible-core==2.19.0b4
```

```suggestion
ansible-core==2.19.0b5
```

Just to make sure, lets test with 2.19.0b5 which has been released recently.

---

I've changed this from 2.19.0b4 to 2.19.0b5.